listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -30,7 +30,7 @@ def get_playing_now(self, user_id):
         if not data:
             return None
         data = ujson.loads(data)
-        data.update({'listened_at': MIN_ID+1})
+        data.update({'listened_at': None})
```

Comment:
Also, we should look into removing MIN_ID altogether from the codebase. Is it used anywhere else?

Reply:
MIN_ID has been used as 'listened_at' value for playing now listens. If we put appropriate checks on timestamp value according to listen type , MIN_ID can be removed from the codebase.